Makefile

```diff
@@ -43,10 +43,16 @@ up:
 	$(DCO_BIN) up -d
 	sleep 3
 
+# Stop the Docker containers
+.PHONY: down
+down:
+	$(DCO_BIN) down
```

Comment:
This does not _stop_ the containers - it deletes them.

Reply:
Thank you, I'm kind of a docker/backend newbie coming from an embedded background. Sometimes I would try to use `make run` and would get an error (don't have an example now), but deleting and rebuilding the containers worked. It could be something specific to my setup though (i'm using this in a docker-in-docker dev container).  Worth keeping this or should I figure out the root cause?

Reply:
I don't think it's worth keeping since `make reset` already drops the containers. We could include `make stop` which runs `$(DCO_BIN) stop` but I don't think that's so useful as I can't imagine you'd need to run that so often, and I don't want to end up creating a make target for every docker-compose command. `make up` is useful since you need to run it every time you want to work on this project.